src/contracts/repository/repository.service.ts

```diff
@@ -24,6 +25,7 @@ export class RepositoryService {
     string,
     LidoAbi | LocatorAbi | SecurityAbi | StakingRouterAbi
   > = {};
+  private cachedDSMPrefixes: Record<string, string> = {};
```

Comment:
Can you add some comment here to explain what is going on?

Reply:
fixed